src/OAuth2/Storage/Mongo.php

```diff
@@ -40,9 +40,6 @@ public function __construct($connection, $config = array())
             $this->db = $m->{$connection['database']};
         }
 
-        // Unix timestamps might get larger than 32 bits,
-        // so let's add native support for 64 bit ints.
-        ini_set('mongo.native_long', 1);
```

Comment:
In the new mongo driver the value of mongo.native_long is set to 1 by default.
Also the setting of this parameter is not supported anymore in 32bit machines and trying to set it generate a fatal error.